virt-install

```diff
@@ -466,6 +466,12 @@ def set_cli_defaults(options, guest):
     if guest.os.is_container():
         return
 
+    if (options.unattended and
+        guest.osinfo.is_windows() and
+        guest.osinfo.get_pre_installable_drivers(guest.os.arch) and
+        guest.osinfo.get_post_installable_drivers(guest.os.arch)):
```

Comment:
It's a bit weird that we are just using these functions as boolean checks. Maybe make these private, but add a more descriptive function like osinfo.supports_unattended_drivers() that hides this logic

Reply:
Also an --unattended option to opt out of this behavior would be useful. Could be a follow up change though